packages/files-ui/src/Components/Modules/FileBrowsers/CreateOrManageSharedFolder.tsx

```diff
@@ -104,23 +266,23 @@ const CreateOrManageSharedFolder = ({ mode, onClose, bucketToEdit }: ICreateOrMa
 
     if (!bucketToEdit) return
 
-    const newWriters = bucketToEdit.writers.map((writer) => ({
-      label: getUserDisplayName(writer),
-      value: writer.uuid || "",
-      data: writer
-    })
-    ) || []
-
     const newReaders = bucketToEdit.readers.map((reader) => ({
       label: getUserDisplayName(reader),
       value: reader.uuid || "",
       data: reader
     })
     ) || []
 
-    onNewUsers(newWriters, "write")
-    onNewUsers(newReaders, "read")
-  }, [bucketToEdit, onNewUsers, onReset])
+    const newWriters = bucketToEdit.writers.map((writer) => ({
+      label: getUserDisplayName(writer),
+      value: writer.uuid || "",
+      data: writer
+    })
+    ) || []
```

Comment:
If the writers array is empty, I'm pretty sure a map returns an empty array anyway. I tested it real quick and nothing crashed
```suggestion
    }))
```